packages/snap-preact-demo/tests/cypress/e2e/tracking/track.cy.js

```diff
@@ -1,382 +1,237 @@
-import { BeaconType, BeaconCategory } from '@searchspring/snap-tracker';
-
-// TODO: update to beacon 2.0
-describe.skip('Tracking', () => {
+describe('Tracking Beacon 2.0', () => {
 	beforeEach(() => {
 		cy.clearCookies();
-
+		cy.clearAllLocalStorage();
 		cy.on('window:before:load', (win) => {
 			win.mergeSnapConfig = {
 				mode: 'production',
 			};
 		});
+	});
+
+	it('tracked shopper login and context data', () => {
+		cy.visit('https://localhost:2222');
 
-		cy.wait(1000);
+		cy.wait(`@beacon2/shopper/login`).then(({ request, response }) => {
+			const { context } = JSON.parse(request.body);
+			expect(context).to.have.property('shopperId').to.be.a('string');
+			expect(context).to.have.property('initiator').to.be.a('string');
+			expect(context).to.have.property('pageLoadId').to.be.a('string');
+			expect(context).to.have.property('pageUrl').to.be.a('string');
+			expect(context).to.have.property('sessionId').to.be.a('string');
+			expect(context).to.have.property('timestamp').to.be.a('string');
+			expect(context).to.have.property('userAgent').to.be.a('string');
+			expect(context).to.have.property('userId').to.be.a('string');
+
+			expect(response.body).to.have.property('success').to.equal(true);
+		});
 	});
-	it('tracked shopper login', () => {
+
+	it('tracked search render, impression, clickthrough', () => {
 		cy.visit('https://localhost:2222');
 
-		cy.waitForBundle().then((searchspring) => {
-			expect(searchspring).to.exist;
+		cy.wait(`@beacon2/search/render`).then(({ request, response }) => {
+			expect(response.body).to.have.property('success').to.equal(true);
+
+			const { context, data } = JSON.parse(request.body);
+			expect(context).to.be.an('object');
+			expect(data).to.have.property('results').to.be.an('array').length.greaterThan(0);
+			expect(data.results[0]).to.have.property('uid').to.be.a('string');
+			expect(data).to.have.property('merchandising').to.be.an('object');
+			expect(data).to.have.property('pagination').to.be.an('object');
+			expect(data).to.have.property('sort').to.be.an('array').length.greaterThan(0);
```

Comment:
This should be an array equal to zero - or undefined, as no sort is applied on base landing page.